src/cmd/apply.ts

```diff
@@ -46,6 +46,7 @@ const prepareValues = async (): Promise<void> => {
   const d = terminal(`cmd:${cmdName}:prepareValues`)
 
   const values = await hfValues()
+  if (!values) throw new Error('No values???')
```

Comment:
Perhaps
`Helmfile does not find any input values`

Reply:
??

Reply:
I am throwing to make a dev in dev mode know he is trying to apply an env dir that was created without values